Documentation/security/network/proxy/envoy.rst

```diff
@@ -10,6 +10,13 @@
 Envoy
 =====
 
+Envoy proxy shipped with Cilium is built with minimal Envoy extensions and custom policy enforcement filters.
+Cilium uses this as its host proxy for enforcing HTTP and other L7 policies as specified in network policies
+for the cluster. Cilium proxy is distributed within the Cilium images.
```

Comment:
Be specific: avoid "this" and "that" unless it's 💯 clear. 

I know "distribution" is overloaded; feel free to replace with a specific noun that works better.
```suggestion
Cilium uses this minimal distribution as its host proxy for enforcing HTTP and other L7 policies as specified in network policies
for the cluster. Cilium proxy is distributed within the Cilium images.
```